runtime/src/iree/schemas/bytecode_module_def.fbs

```diff
@@ -246,7 +246,7 @@ table BytecodeModuleDef {
   // Features required to load and run bytecode from the module.
   // Some functions may have additional requirements for when there are multiple
   // variants selected based on runtime feature detection.
-  requirements:FeatureBits = 0;
+  requirements:FeatureBits = EXT_F32;
```

Comment:
What was the intent here, @benvanik ?
```
  // 32-bit floating point extension.
  EXT_F32 = 0,  // 1u << 0
  // 64-bit floating point extension.
  EXT_F64 = 1,  // 1u << 1
```

Isn't `1u << 0 == 1`, and `1u << 1 == 2`? Is the intent for this to default to "no features", i.e. no f64 _or_ f32? Or is f32 included by default?

Reply:
Ah: https://github.com/google/flatbuffers/blob/master/docs/source/Schemas.md#attributes

> `bit_flags` (on an unsigned enum): the values of this field indicate bits, meaning that any unsigned value N specified in the schema will end up representing 1<<N, or if you don't specify values at all, you'll get the sequence 1, 2, 4, 8, ...

So the comments are correct